src/audit/routes/system.py

```diff
@@ -1,10 +1,74 @@
 from fastapi import APIRouter, FastAPI, Request, Depends
+from typing import Union, get_args, get_origin
 
 from ..db import DataAccessLayer, get_data_access_layer
 
+from ..models import CreateLoginLogInput, CreatePresignedUrlLogInput
 
 router = APIRouter()
 
+# Increment versions on schema/model change.
+CURRENT_SCHEMA_VERSIONS = {
```

Comment:
wondering how we could enforce that this is updated when the model classes are updated

Reply:
ooh i like it!

Reply:
Yeah i don't know how to enforce it 100%, maybe it would be a bit overengineered anyway. Let's just use the fingerprint and the test can instruct devs to update the version+fingerprint in `CURRENT_SCHEMA_VERSIONS` on failure

Reply:
The comment above `CURRENT_SCHEMA_VERSIONS` can also be updated with a warning that BOTH have to be updated. So reviewers could catch it too